lib/PostMessageTransport.js

```diff
@@ -83,6 +95,13 @@ define(function (require, exports, module) {
             PostMessageTransportRemote +
             "</script>\n";
     }
+
+    function reload() {
+        var _launcher = Launcher.getCurrentInstance();
+        var _liveDoc = LiveDevMultiBrowser._getCurrentLiveDoc();
```

Comment:
Remove the `_` from both of these variables (lines 100, 101) since they are local to this function vs. private, globals.